### ISSUE TITLE
Delete behaviour broken on qbitorrent 4.5.x

### ISSUE THREAD

**Issue by ethw** [jerrymakesjelly/autoremove-torrents#170]
Using a qbittorrent 4.5.x client results in an `HTTP 405` error when running an autoremove-torrents task that would delete a torrent. This is because qbittorrent 4.5+ restricted the delete api to be `POST` request only, and I guess this tool is not using `POST`. [source](https://github.com/qbittorrent/qBittorrent/issues/18097)

I switched to using qbittorrent 4.2.x and autoremove-torrents worked perfectly.

Could this tool please be updated to use `POST` for the delete request?

Here's a log where I got the error, though it doesn't look very useful to me.

```
Mon, 13 Mar 2023 22:21:24 autoremovetorrents.main INFO: Auto Remove Torrents 1.5.4
Mon, 13 Mar 2023 22:21:24 autoremovetorrents.main INFO: Loading configurations...
Mon, 13 Mar 2023 22:21:24 autoremovetorrents.main INFO: Found 1 task(s) in the file.
Mon, 13 Mar 2023 22:21:24 autoremovetorrents.task INFO: Running task 'qbittorrent'...
Mon, 13 Mar 2023 22:21:24 autoremovetorrents.task INFO: Logging in...
Mon, 13 Mar 2023 22:21:24 autoremovetorrents.task INFO: Login successfully. The client is qBittorrent v4.5.2.
Mon, 13 Mar 2023 22:21:24 autoremovetorrents.task INFO: WebUI API version: 2.8.19 (v2)
Mon, 13 Mar 2023 22:21:24 autoremovetorrents.task INFO: Status reported by the client:
        Download Speed: 0.00B/s Total: 0.00B
        Upload Speed: 0.00B/s   Total: 0.00B
Mon, 13 Mar 2023 22:21:24 autoremovetorrents.task INFO: Getting all the torrents...
Mon, 13 Mar 2023 22:21:24 autoremovetorrents.task INFO: Found 1 torrent(s) in the client.
Mon, 13 Mar 2023 22:21:24 autoremovetorrents.strategy INFO: Running strategy idk...
WARNING: Couldn't write lextab module 'lextab'. [Errno 20] Not a directory: '/usr/local/lib/python3.10/dist-packages/autoremove_torrents-1.5.4-py3.10.egg/autoremovetorrents/lextab.py'
Mon, 13 Mar 2023 22:21:24 autoremovetorrents.strategy INFO: Total: 1 torrent(s). 1 torrent(s) can be removed.
Mon, 13 Mar 2023 22:21:24 autoremovetorrents.strategy INFO: To be deleted:
Mon, 13 Mar 2023 22:21:24 autoremovetorrents.strategy INFO: School.Spirits.2023.S01E02.720p.AMZN.WEBRip.DDP5.1.x264-PlayWEB[rartv]
        Progress:100.00%        Size:1.45GiB    Ratio:0.125     Total Uploaded:187.00MiB
        Seeder(connected/total):0/35    Leecher(connected/total):0/11   Status:Uploading
        Download Speed:0.00B/s(Avg.:19.35MiB/s) Upload Speed:0.00B/s(Avg.:158.12KiB/s)
        Create Time:2023-03-13 22:01:06 Seeding Time:0d 00:18:54        Downloading Time:(Not Provided) Last Activity:0d 00:03:51
        Category:manual Tracker:** [DHT] **,** [PeX] **,** [LSD] **,tracker.trackerfix.com,9.rarbg.me,9.rarbg.to,tracker.tallpenguin.org,tracker.thinelephant.org
Mon, 13 Mar 2023 22:21:24 autoremovetorrents.task ERROR: The torrent School.Spirits.2023.S01E02.720p.AMZN.WEBRip.DDP5.1.x264-PlayWEB[rartv] and its data cannot be removed. Reason: The server responses HTTP 405.
```

**Comment by ethw** [comment]
Duplicate https://github.com/jerrymakesjelly/autoremove-torrents/issues/157

**Comment by ethw** [comment]
Duplicate https://github.com/jerrymakesjelly/autoremove-torrents/issues/157

and fixed in https://github.com/jerrymakesjelly/autoremove-torrents/pull/160

**Comment by ethw** [comment]
Duplicate of #157
and fixed in #160
